diff --git a/src/community/infomap/infomap.cc b/src/community/infomap/infomap.cc
index a4cd217f6c..94ca66fc8f 100644
--- a/src/community/infomap/infomap.cc
+++ b/src/community/infomap/infomap.cc
@@ -29,15 +29,16 @@
    homePage: http://www.irit.fr/~Emmanuel.Navarro/
 */
 
-#include <cmath>
+
 #include "igraph_interface.h"
 #include "igraph_community.h"
 #include "core/interruption.h"
 
-
 #include "infomap_Node.h"
 #include "infomap_Greedy.h"
 
+#include <math.h>
+
 /****************************************************************************/
 int infomap_partition(FlowGraph * fgraph, bool rcall) {
     Greedy * greedy;
